handbook/communication/content_guidelines/style_and_mechanics.md

```diff
@@ -798,13 +798,13 @@ Refer to the natural noun of the product or feature directly.
 <div class="item yes">
 <h5>Yes</h5>
 <ul>
-<li>Batch Changes are available.</li>
+<li>Extensions are available.</li>
```

Comment:
it may be helpful to include both an example of a product and feature (so include Batch Changes and Extensions)